tests/fv/fv_test.go

```diff
@@ -81,6 +81,101 @@ var _ = Describe("PolicyController", func() {
 		apiserver.Stop()
 	})
 
+	Context("nodes", func() {
+		// It("should delete a node that exists at start of day", func() {})
+		// It("should delete nodes that nodeRef unexisting node in k8s"), func() {})
+		// It("should clean up dead data for nodes"
+		It("should be removed in response to a k8s node delete", func() {
```

Comment:
nit: the "should" clauses here are worded strangely. I think they are assuming "It" is the node object, when I think "It" should be the controller.

e.g. for this one,  "It should delete Calico nodes in response to deleting a k8s node"

Reply:
I thought this was correct because these are all within `Context("nodes" )`

Reply:
Yeah perhaps, I hadn't seen that.

However, I'd also suggest tweaking the context to "Node controller" to match the other contexts in this file